ebb_lint/test/test_flake8.py

```diff
@@ -65,7 +65,7 @@ def replacement(match):
 @pytest.fixture(autouse=True)
 def scan_for_checkers(request):
     parser, options_hooks = get_parser()
-    args_marker = request.node.get_marker('flake8_args')
+    args_marker = request.node.get_closest_marker('flake8_args')
```

Comment:
To avoid a warning:

    MarkInfo objects are deprecated as they contain merged marks which are hard to deal with correctly. Please use node.get_closest_marker(name) or node.iter_markers(name).